### DISCUSSION TITLE
Bootscore Commerce child theme - change product thumbnail look

### DISCUSSION THREAD

**Discussion by noyloza** [bootscore/bootscore#330] [Q&A] [answered]
I'm trying to change the way the thumbnails look so the images do not get cropped. I would prefer the actual scale of the image comes out, or at least it doesn't get cropped. Is this possible? Here is my link: https://patrick-diaz.com/buy/

I'm just trying to prevent cropping on these product images.

My other question is what file controls the actual single product info? I want to push the description from under the image on the single product page, over to the right of the product images on the right side.

**Comment by crftwrk** [comment, marked as answer]
Please ask only one question per topic to keep this forum clean. Thank you.

### Woo images

Click in the admin bar Customize > WooCommerce > Product Images and set to Uncropped

<img width="293" alt="Bildschirmfoto 2022-12-19 um 17 43 09" src="https://user-images.githubusercontent.com/51531217/208476179-f7758f25-bbe9-4079-bc6b-a112a38a440d.png">

### Product description

Simply use the Product short description in product backend. This is shown on the right side instead below.

Does this help?

**Reply by noyloza** [reply]
yes thank you very much

**Reply by crftwrk** [reply]
You‘re welcome 👍

**Comment by noyloza** [comment]
Is it possible to not have a main description, and only have the short? When I do this on my product, it seems to break the page: https://patrick-diaz.com/buy/product/collection-of-a-texas-lady/

The page will work if I have content in both descriptions, but not if it is only short. Am I missing a file possibly?

**Reply by crftwrk** [reply]
This was fixed in an earlier update https://github.com/bootscore/bootscore/issues/118. 

You are using 5.1.3.2 which is outdated. Use 5.2.3.0  https://github.com/bootscore/bootscore/releases and bootcommerce-child 5.2.0.0 https://github.com/bootscore/bootcommerce-child/releases. Note that there are breaking changes, you must use new child as well.

Let me know if that helps

**Reply by crftwrk** [reply]
This was added in 5.2.0.0 to fix that https://github.com/bootscore/bootscore/pull/121

```css
.related.products {
  clear: both;
}
```

**Reply by noyloza** [reply]
Thank you. I upgraded everything and it's much better!
